src/sbd-inquisitor.c

```diff
@@ -194,6 +194,7 @@ int check_all_dead(void)
 
 	for (s = servants_leader; s; s = s->next) {
 		if (s->pid != 0) {
+			memset(&svalue, 0, sizeof(svalue));
 			r = sigqueue(s->pid, 0, svalue);
```

Comment:
Since svalue is only ever needed for sigqueue(), a small sigqueue() wrapper would isolate svalue and memset() (making the problem less likely to be reintroduced in the future if another sigqueue() call is needed)